tests/helpers/acceptance.js

```diff
@@ -8,6 +8,12 @@ var Promise           = require('../../lib/ext/promise');
 var root = path.resolve(__dirname, '..', '..');
 
 var PackageCache = require('../../tests/helpers/package-cache');
+var CommandGenerator = require('../../tests/helpers/command-generator');
+
+// We must link `ember-cli`.
+var npm = new CommandGenerator(require.resolve('npm/bin/npm-cli.js'));
+npm.invoke('link', { cwd: root });
```

Comment:
Would it be possible to instead defer this to the `PackageCache`?  I'd love to be able to test different `ember-cli` builds at the same time (or test locally without completely borking my ability to develop normal apps / addons)...

What if we had a `links` in the `PackageCache` manifest as an array of objects (with `name` and `destination` keys)?